README.md

```diff
@@ -25,17 +34,31 @@ Use the appropriate repos for your distribution and install using the usual tool
 
 Get the source by cloning this repo and do a pip install:
 
+As pip will have to copy files to /etc/cloud-info-provider directory, the
+installation user should be able to write to it, so it is recommended to create
+it before using pip.
+
+``` sh
+sudo mkdir /etc/cloud-info-provider
+sudo chgrp you_user /etc/cloud-info-provider
+sudo chmod g+rwx /etc/cloud-info-provider
+```
+
 ```
-git clone https://github.com/EGI-FCTF/BDIIscripts
-cd BDIIscripts 
+git clone https://github.com/EGI-FCTF/cloud-bdii-provider
+cd cloud-bdii-provider 
```

Comment:
This should be changed to cloud-info-provider, right? When the PR is merged, we can rename the repository (in Github) itself.